index.bs

```diff
@@ -185,6 +188,30 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
     </section>
 
     <section>
+      <h4 id='mime-type'>MIME types</h4>
+
+      <p>
+        In the context of this specification, a MIME type is also called content
+        type. A <dfn>valid media MIME type</dfn> is a string that is a <a>valid
+        MIME type</a> per [[mimesniff]] and must have only one parameter that is
+        named <code>codecs</code> which contains only one value that is not the
```

Comment:
We'd like for the codecs string to describe just one codec; no comma separated values. Perhaps something like this helps clarify:

"...and must have only one parameter that is named codecs with a value describing a single media codec."